Tests/TuistLoaderIntegrationTests/ProjectDescriptionHelpers/ProjectDescriptionHelpersBuilderIntegrationTests.swift

```diff
@@ -38,13 +38,13 @@ final class ProjectDescriptionHelpersBuilderIntegrationTests: TuistTestCase {
         let searchPaths = ProjectDescriptionSearchPaths.paths(for: projectDescriptionPath)
 
         // When
-        let paths = try (0 ..< 3).map { _ in try subject.build(at: path, projectDescriptionSearchPaths: searchPaths) }
+        let paths = try (0 ..< 3).map { _ in try subject.build(at: path, projectDescriptionSearchPaths: searchPaths, customProjectDescriptionHelpers: []) }
 
         // Then
         XCTAssertEqual(Set(paths).count, 1)
         XCTAssertNotNil(FileHandler.shared.glob(path, glob: "*/*/ProjectDescriptionHelpers.swiftmodule").first)
         XCTAssertNotNil(FileHandler.shared.glob(path, glob: "*/*/libProjectDescriptionHelpers.dylib").first)
         XCTAssertNotNil(FileHandler.shared.glob(path, glob: "*/*/ProjectDescriptionHelpers.swiftdoc").first)
-        XCTAssertTrue(FileHandler.shared.exists(paths.first!!))
+        XCTAssertTrue(FileHandler.shared.exists(paths[0][0].path))
```

Comment:
indexing is basically force unwrapping, so this code is not much safer (although maybe a little bit readable?) - I'd use `XCTUnwrap` here